cmd/metrics-prometheus-collector/integration/testdata/over-max-extension-labels/input_collector.yaml

```diff
@@ -217,6 +217,7 @@ resources:
   "items": "items"
 preComputeMetrics:
   enabled: true
+  frequency: "1s"
```

Comment:
should we add this to the configmap under config/, or just take the default value?

Reply:
Either is fine.  I'll add it just so it is documented better.

Reply:
Done.  PTAL